Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -164,5 +178,20 @@ public void updateMotorTelemetry() {
         leftDriveSlave.updateTelemetryProperties();
         rightDrive.updateTelemetryProperties();
         rightDriveSlave.updateTelemetryProperties();
+        
+        leftDistanceProp.set(leftDrive.getPosition());
+        rightDistanceProp.set(rightDrive.getPosition());
+    }
+    
+    public double getDistance() {
+        return convertToInches(leftDrive.getPosition() - startPosition.get());
```

Comment:
`startPosition` is already in inches, while `getPosition` returns ticks... and then it's put through an inch conversion. Something needs to be moved.